core/templates/pages/preferences-page/preferences-page.component.ts

```diff
@@ -66,10 +66,14 @@ angular.module('oppia').component('preferencesPage', {
         return UrlInterpolationService.getStaticImageUrl(imagePath);
       };
       var _saveDataItem = function(updateType, data) {
+        ctrl.allowreload = false;
```

Comment:
Do we need to attach `allowreload` to ctrl?

Reply:
yes otherwise it shows an error

Reply:
Can you share the log and have you tried resolving the issue? (Asking this as we only attach function/variables to ctrl which needs to be used in the template file.)

Reply:
Converted it to `var allowreload = true;`. Resolved it.

Reply:
Re the renaming, I would prefer to have it `preventReload` (as our feature is directly around prevent reloading, reloading is allowed by default).